### PR TITLE
Added endpoints to list sessions and logoff a session

### PR DESCRIPTION
**Pull request by antoineduban**
Nanocloud and plaza now both have corresponding endpoints to list all active windows session for the admin, and the current one ( if there is one ) for a regular user. Additionally , the logoff endpoint will be used to logoff the user who sent the request from windows.

### CODE REVIEW

**Comment by Gentux** [comment]
Great pull request !

Tests are missing though, can you add it ?

File: nanocloud/routes/sessions/sessions.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				hash{
+					"detail": err.Error(),
+				},
+			},
+		})
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Error(err)
```
Comment (dynamiccast) [inline nanocloud/routes/sessions/sessions.go:34]: Do we want that ?

Comment (antoineduban) [inline nanocloud/routes/sessions/sessions.go:34]: Yes

File: tests/api/nanotest.js [hunk 2]
```diff
@@ -116,15 +125,16 @@ var nano = {
         var validator = new JSONAPIValidator();
         var valid = true;
 
+        var tmp = this.response.data;
```
Comment (dynamiccast) [inline tests/api/nanotest.js]: We should directly use  this.response.data

File: tests/api/nanotest.js [hunk 3]
```diff
@@ -116,15 +125,16 @@ var nano = {
...
         try {
-          validator.validate(this.response.data);
+          validator.validate(tmp);
         } catch (e) {
             valid = false;
         }
 
-        if (valid && !validator.isValid(this.response.data)) {
+/*        if (valid && !validator.isValid(this.response.data)) {
```
Comment (dynamiccast) [inline tests/api/nanotest.js]: Commented code here

**Pull request merged**